content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -73,18 +73,18 @@ The biggest difference to previous stages in our pipeline is the new `metrics`
 section. Metrics files contain scalar values (e.g. `AUC`) to compare across
 iterations.
 
-> With `cache: false`, DVC skips caching the output, as we want these JSON
-> metrics and plots files to be versioned by Git.
+<admon type="info">
+
+With `cache: false`, DVC skips caching the output, as we want these JSON metrics
+files to be versioned by Git.
+
+</admon>
 
 </details>
 
-[`evaluate.py`](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py)
-writes the model's
-[ROC-AUC](https://scikit-learn.org/stable/modules/model_evaluation.html#receiver-operating-characteristic-roc)
-and
-[average precision](https://scikit-learn.org/stable/modules/model_evaluation.html#precision-recall-and-f-measures)
-for both train and test datasets to `eval/live/metrics.json`, which in turn is
-marked as a `metrics` file with `-M`. Its contents are:
+[`evaluate.py`] writes the model's [ROC-AUC] and [average precision] (for both
+datasets) to `eval/live/metrics.json` (which was marked as a [metrics file] with
+`-M` above):
```

Comment:
2 questions:
1. Do we mention train and test datasets above? How will someone know to what "both datasets" refers?
2. What's the rationale for removing links to background info on those metrics?

Reply:
The 2 sets are shown in the example itself but I removed that parenthesis anyway (not needed). The metrics are still linked, to the specific framework implementation docs. Those docs link to more background info, so having 2 sets of different links for metrics here seemed unnecessary.

Reply:
Okay, I missed that the background info was linked in the metrics already. Agreed that it's not needed in both places, plus to me it looks like these plots links should go to the actual files in the repo, so I'm good with removing them.

Reply:
I tried linking the file path to the repo but it's too much (too many links in that sentence already and since the code block below is the same). But feel free to revise the linking of that and other paragraphs that also mention file paths in the repo if you prefer.